### PR TITLE
WIP: Windup 2149

### PR DESCRIPTION
**Pull request by jeichler**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -163,9 +163,37 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-compiler-plugin</artifactId>
-                    <version>3.2</version>
+                    <version>3.8.0</version>
```
Comment (jeichler) [inline pom.xml]: this is not yet done, but works for now

**Comment by windupgithubbot1** [comment]
Can one of the admins verify this patch?

**Comment by windupgithubbot1** [comment]
Can one of the admins verify this patch?

**Comment by windupgithubbot1** [comment]
Can one of the admins verify this patch?

**Comment by windupgithubbot1** [comment]
Can one of the admins verify this patch?

**Comment by windupgithubbot1** [comment]
Can one of the admins verify this patch?

**Pull request closed without merging**